libbeat/common/jsontransform/jsonhelper.go

```diff
@@ -49,3 +49,7 @@ func WriteJSONKeys(event common.MapStr, keys map[string]interface{}, overwriteKe
 
 	}
 }
+
+func CreateJSONError(message string) common.MapStr {
```

Comment:
I exported it because I also use it in the json reader. But I can also just copy / paste the code to the json reader.